metadata/vswitch.xml

```diff
@@ -7,47 +7,47 @@
 		<option name="binding_left" type="activator">
 			<_short>Left</_short>
 			<_long>Switches to workspace left with the specified activator.</_long>
-			<default>&lt;super&gt; KEY_LEFT</default>
```

Comment:
I don't see the need to change the KEY_[DIRECTION] defaults for vswitch. The arrow keys are what a user is more likely to expect as a default.

Reply:
It is used for the grid.
`HJKL` is used for navigating workspaces and the arrow keys and keypad for the grid.

https://github.com/WayfireWM/wayfire/wiki/Tutorial#default-key-bindings